include/fmt/chrono.h

```diff
@@ -664,6 +664,30 @@ enum class numeric_system {
   alternative
 };
 
+// Glibc extensions for formatting numeric values.
+enum class pad_type {
+  unspecified,
```

Comment:
Do we need unspecified? It looks like it's always handled as zero and can be removed.

Reply:
Based on the current codebase, yes.